### DISCUSSION TITLE
NGワードのエラーにも関わらず、「Qiitaのアクセストークンが正しいかご確認ください」というエラー文になっている

### DISCUSSION THREAD

**Discussion by porcelain888** [increments/qiita-discussions#723] [要望・アイデア・不具合]
## エラーの原因と解析
ブロックチェーンの記事を書いていて、Binance Smart Chain と記載したところ、Binance がNGワードになっていた模様

## 困ったこと
エラー文が「Qiitaのアクセストークンが正しいかご確認ください」となり、アクセストークンを変更しても一向に解決できなかった

## 修正案
「NGワードが含まれています」というエラー文に変更してほしい

**Comment by tanaka-asami415** [comment]
@porcelain888 
ご不便をおかけして申し訳ありません。
恐らくCLIのご利用時かと思うのですが、エラーメッセージが分かりにくいというご意見は他にもいただいているため、改善を検討しております。
また、`Binance` が含まれる場合に投稿できない不具合がありましたので、修正いたしました🙇

**Comment by wataru86** [comment]
@porcelain888

お待たせしました。
publish コマンド実行時のエラーメッセージを修正し、「記事ファイルに不備がないかご確認ください」というメッセージも表示するようにしました。

現状では API からエラーの原因を取得できないため、詳細なエラーを表示することはできませんが、API の改善と併せて将来的に検討したいと思っています！
